lib/actions/assign.js

```diff
@@ -35,7 +42,7 @@ class Assign extends Action {
   async afterValidate (context, settings, name, results) {
     const payload = this.getPayload(context)
     const issueNumber = payload.number
-    const assignees = settings.assignees
+    const assignees = await Promise.all(settings.assignees.map(assignee => mapSpecialAssignee(payload, assignee)))
```

Comment:
if `mapSpecialAssignee` is not async then you could remove `await Promise.all` and make it simpler